### ISSUE TITLE
Issue parsing cloudformation ListTypes in multiple regions

### ISSUE THREAD

**Issue by michaeljoy** [JohannesEbke/aws_list_all#33]
```Service: cloudformation               | Region: us-east-1       | Operation: ListTypes
Traceback (most recent call last):
  File "/usr/local/bin/aws-list-all", line 8, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.7/site-packages/aws_list_all/__main__.py", line 142, in main
    do_query(services, args.region, args.operation, verbose=args.verbose or 0, parallel=args.parallel)
  File "/usr/local/lib/python3.7/site-packages/aws_list_all/query.py", line 206, in do_query
    print('Service: {: <28} | Region: {:<15} | Operation: {}'.format(service, region, operation))
TypeError: unsupported format string passed to NoneType.__format__
```

**Comment by michaeljoy** [comment]
Running with --parallel 1 also does not change behavior, nor does changing region to us-east-1

**Comment by SleepyBrett** [comment]
same problem.

**Comment by JohannesEbke** [comment]
Should be fixed in v0.9
